KMB/make_KMB_info.py

```diff
@@ -154,22 +197,21 @@ def generate_content_cats(self, item):
         :param item: the KMBItem to analyse
         :return: list of categories (without "Category:" prefix)
         """
-        cats = []
+        cats = item.content_cats
 
         # depicted
         if item.fmis:
-            cats.append('Archaeological monuments in %s' % item.landskap)
-            cats.append('Archaeological monuments in %s County' % item.lan)
+            cats.add('Archaeological monuments in %s' % item.landskap)
+            cats.add('Archaeological monuments in %s County' % item.lan)
         if item.bbr:
-            cats.append('Listed buildings in Sweden')
+            cats.add('Listed buildings in Sweden')
```

Comment:
I remember you mentioning these will be put into correct subcats by a bot, is that right?